packages/salesforcedx-vscode-apex/src/views/testOutlineProvider.ts

```diff
@@ -152,19 +155,11 @@ export class ApexTestOutlineProvider
   }
 
   private getJSONFileOutput(fullFolderName: string): FullTestResult {
-    const files = fs.readdirSync(fullFolderName);
-    let fileName = files[0];
-    for (const file of files) {
-      if (
-        file !== 'test-result-codecoverage.json' &&
-        ospath.extname(file) === '.json' &&
-        file.startsWith('test-result')
-      ) {
-        fileName = file;
-      }
-    }
+    const testIdFile = path.join(fullFolderName, 'test-run-id.txt');
```

Comment:
Let's change the name of these variables to testRunIdFile (from testIdFile) & testRunId (from testId). It'll map directly to the name of the file we are working with and would be easier to read the code.

Reply:
Okay makes sense